sktime/forecasting/compose/tests/test_multiplex.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Tests for MultiplexForecaster and associated dunders."""
+
+__author__ = ["aiwalter", "miraep8"]
+
+from sktime.datasets import load_shampoo_sales
+from sktime.forecasting.all import (
```

Comment:
better not use `forecasting.all`, it has not been used in a while, and who knows what will happen with it.

In case you want to help fix the `all` import, have a look at this issue: https://github.com/alan-turing-institute/sktime/issues/1928 (until then I would recommend to not use it)

Reply:
Sure, will fix this to be more specific with the imports!  Thanks for the heads up :)

Reply:
I actually think I may have copied that from the MutliplexForecaster example haha, so will be sure to go and fix it there as well. :)

Reply:
Actually - if it all right with you I might actually fix the MultiplexForecaster input example in my upcoming `|` dunder PR.  As it stands it is all mixed up with those changes I was working on but didn't necessarily want to be a part of this PR.

Reply:
sure, makes sense - generally it´s a good idea to keep PR localized (by file or by content)